test/test_twobytwo_wTaxes.jl

```diff
@@ -236,6 +236,12 @@
 
     
     set_value!(otax, 0.9)
+
+    ### Bug if not set. Not certain what the issue is.
+    ### This only breaks if run in this order, if you set otax to .9 before
+    ### solving with otax = .5 then it's fine. 
+    fix(RA,462)
+
```

Comment:
I don't know what the issue is here. I'm going to discuss this with Tom on Tuesday or Wednesday